code/parse/sexp.cpp

```diff
@@ -14220,7 +14044,7 @@ void sexp_ships_stealthy(int n, bool stealthy)
 // Goober5000
 void sexp_friendly_stealth_invisible(int n, bool invisible)
 {
-	sexp_deal_with_ship_flag(n, true, 0, 0, 0, SF2_FRIENDLY_STEALTH_INVIS, 0, P_SF2_FRIENDLY_STEALTH_INVIS, invisible, true);
+	sexp_deal_with_ship_flag(n, true, Object::Object_Flags::NUM_VALUES, Ship::Ship_Flags::Stealth, Mission::Parse_Object_Flags::SF_Friendly_stealth_invis, invisible, true);
```

Comment:
The old code used `SF2_FRIENDLY_STEALTH_INVIS`. Is `Ship::Ship_Flags::Stealth` the same flag?